specs/experimental/admin-and-auth.md

```diff
@@ -25,21 +28,131 @@
 
 <!-- END doctoc generated TOC please keep comment here to allow auto update -->
 
-The Security Council (at
-[eth:0xc2819DC788505Aac350142A7A707BF9D03E3Bd03](https://etherscan.io/address/0xc2819DC788505Aac350142A7A707BF9D03E3Bd03))
-uses a specially extended Safe multisig contract to provide additional security guarantees on top of
-those provided by the Safe contract.
+## Overview
+
+Authorization of administrative actions in the Superchain are managed by a system of multisignature
+ Safe contracts, as well as custom extensions to those Safe contracts which provide additional
+functionality. This document describes the system of Safe's and their purposes.
+
+## List of Safes
+
+This list outlines the various Safes, including mainnet addresses, thresholds, ownership, and any
+extensions.
+
+1. **[The ProxyAdminOwner
+   Safe](https://etherscan.io/address/0x5a0Aae59D09fccBdDb6C6CcEB07B7279367C3d2A):** The name of
+    this Safe is slightly misleading. While it does control the `ProxyAdmin` contract, can therefore
+   upgrade contracts in the system, more generally it is in charge of _safety_, meaning it should
+   control any action which has an impact on the determination of a valid L2 state, or the custody
+   of bridged assets. This includes but is not limited to upgrading L1 contracts, and modifying the
+   implementation of the dispute game.
```

Comment:
Hmm, I think we should avoid the specs becoming the source of truth for the current configuration and roles in the system (current config should be in docs, and roles should be a separate specs page, similar to [this docs page](https://docs.optimism.io/chain/security/privileged-roles)), and instead have them focus on the extensions and how they're intended to be used with Safes. The "List of Safes" content seems better suited for docs, but the rest of the spec seems like the right content

Reply:
Actually, for Safe's owned by Safes (the POA and Guardian Safe) it still seems necessary to keep the threshold and ownership info. WDYT?

Reply:
> Hmm, I think we should avoid the specs becoming the source of truth for the current configuration and roles in the system (current config should be in docs, and roles should be a separate specs page, similar to [this docs page](https://docs.optimism.io/chain/security/privileged-roles)), and instead have them focus on the extensions and how they're intended to be used with Safes. The "List of Safes" content seems better suited for docs, but the rest of the spec seems like the right content

We've agreed to: 
1. assume stage 1 is a requirement for implementing the protocol
2. create a new specs doc dedicated to that requirement, it will include the list of safes and diagram
3. have this document focus on the Safe extensions.